optaplanner-core/src/build/revapi-config.json

```diff
@@ -224,6 +224,70 @@
           "elementKind": "method",
           "justification": "Introduced QuadStreams"
         },
+        {
+          "code": "java.method.numberOfParametersChanged",
+          "old": "method <GroupKeyA_, GroupKeyB_, ResultContainer_, Result_> org.optaplanner.core.api.score.stream.tri.TriConstraintStream<GroupKeyA_, GroupKeyB_, Result_> org.optaplanner.core.api.score.stream.bi.BiConstraintStream<A, B>::groupBy(java.util.function.BiFunction<A, B, GroupKeyA_>, java.util.function.BiFunction<A, B, GroupKeyB_>, org.optaplanner.core.api.score.stream.bi.BiConstraintCollector<A, B, ResultContainer_, Result_>)",
+          "new": "method <GroupKey_, ResultContainer_, Result_> org.optaplanner.core.api.score.stream.bi.BiConstraintStream<GroupKey_, Result_> org.optaplanner.core.api.score.stream.bi.BiConstraintStream<A, B>::groupBy(java.util.function.BiFunction<A, B, GroupKey_>, org.optaplanner.core.api.score.stream.bi.BiConstraintCollector<A, B, ResultContainer_, Result_>)",
```

Comment:
@ge0ffrey It appears to me that these Revapi changes make no sense.

This one complains I changed something about TriStreams. I reviewed the history for both this PR and the previous (merged) PR and there is nothing that does anything with TriStreams.

Do you have any idea of what this could mean? Something I'm missing?

Reply:
No, this entire thing looks strange to me